Misc/NEWS.d/next/Documentation/2018-03-05-08-33-00.bpo-28955.-OSvYJ.rst

```diff
@@ -0,0 +1 @@
+Issue 28955 - Clarified numeric comparisons with NaN. Patch by Tony Flury
```

Comment:
We don't require news entries for trivial documentation changes. You can delete this file.